.tflint-msk.hcl

```diff
@@ -10,5 +12,5 @@ rule "terraform_standard_module_structure" {
 
 # Include module calls
 config {
-  module = true
+  call_module_type = "all"
```

Comment:
this seems to require tflint v0.50.0+ https://github.com/terraform-linters/tflint/releases/tag/v0.50.0 but I don't think we version this at all (it needs to be installed independently from `pre-commit`, maybe worth noting somewhere that we only support running the latest version of `tflint`

Reply:
We mention installing it here: https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/CONTRIBUTING.md. Maybe worth moving it directly to readme, as it might be missed